src/Draco.ProjectSystem/Project.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Execution;
+using MSBuildProject = Microsoft.Build.Evaluation.Project;
+using MSBuildProjectInstance = Microsoft.Build.Execution.ProjectInstance;
+
+namespace Draco.ProjectSystem;
+
+/// <summary>
+/// Represents a single project in a workspace.
+/// </summary>
+public sealed class Project
+{
+    /// <summary>
+    /// The project file.
+    /// </summary>
+    public FileInfo ProjectFile { get; }
+
+    /// <summary>
+    /// THe workspace the project belongs to.
+    /// </summary>
+    public Workspace Workspace { get; }
+
+    /// <summary>
+    /// The target framework of the project.
+    /// </summary>
+    public string? TargetFramework
```

Comment:
How is this related to the TargetFramework in a csproj ? 
There is a TargetFramework, and TargetFrameworks in csproj.

Reply:
It's the target FW we choose for the design time build